websites/D/Desu-Online/metadata.json

```diff
@@ -20,5 +20,7 @@
 		"anime"
 	],
 	"iframe": true,
-	"iFrameRegExp": ".*"
+	"iFrameRegExp": ".*",
+	"iFrameMatches": ["*://*.cda.pl/*", "*://*.google.com/*", "*://*.mega.nz/*", "*://*.dailymotion.com/*", "*://*.sibnet.ru/*", "*://*.fileditchstuff.me/*", "*://*.yourupload.com/*", "*://*.vk.com/*", "*://*.wolfstream.tv/*", "*://*.ok.ru/*"],
```

Comment:
This doesnt match iframeregexp

Reply:
![image](https://github.com/PreMiD/Presences/assets/42370647/afa66ef0-e2b1-4035-b972-13a6fc30f672)
i was sure so i changed to this ^

Reply:
nvm, im gonna do this in regexp